### ISSUE TITLE
MorphTo relationships do not work when creating an object

### ISSUE THREAD

**Issue by Hannoma** [laravel/nova-issues#3981]
- Laravel Version: 8.83.8
- Nova Version: 4.1.1
- PHP Version: 8.1.4

### Description:
When creating a resource with a morphTo relationship there are no values included in the post request.
![image](https://user-images.githubusercontent.com/7955676/163247163-1fd14fc9-ed89-4ed4-bc6e-046e7bcfc857.png)

When updating, there is no error.

### Detailed steps to reproduce the issue on a fresh Nova installation:
`Booking.php`
```php
public function fields(NovaRequest $request): array
    {
        return [
            ID::make('ID')
                ->hideFromIndex(),

            MorphTo::make('Bookable')->types([
                Room::class,
                Vehicle::class,
            ]),

            // More fields here
        ];
    }
```

`Room.php`
```php
public function fields(NovaRequest $request): array
    {
        return [
            ID::make('ID')
                ->hideFromIndex(),

            // More fields here

            MorphMany::make('Bookings'),
        ];
    }
```

`Vehicle.php`
```php
public function fields(NovaRequest $request): array
    {
        return [
            ID::make('ID')
                ->hideFromIndex(),

            // More fields here

            MorphMany::make('Bookings'),
        ];
    }
```

**Comment by crynobone** [comment]
This should already be fixed, can you upgrade to 4.2.2 and verify again.

**Comment by Hannoma** [comment]
Found the issue. Forgot to run `php artisan nova:publish`. Sorry for causing trouble
